I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/gui.rpy", line 362, in script
    define config.nvl_show_transition = Dissolve(0.5)
  File "renpy/common/000namespaces.rpy", line 34, in set
    setattr(self.nso, name, value)
Exception: config.nvl_show_transition is not a known configuration variable.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/gui.rpy", line 362, in script
    define config.nvl_show_transition = Dissolve(0.5)
  File "/nix/store/jrq8zj3xplxdf1fhmxhdb43pl7pq2agg-renpy-8.3.7.25031702/share/renpy/renpy/ast.py", line 2248, in execute
    self.set()
  File "/nix/store/jrq8zj3xplxdf1fhmxhdb43pl7pq2agg-renpy-8.3.7.25031702/share/renpy/renpy/ast.py", line 2266, in set
    ns.set(self.varname, value)
  File "renpy/common/000namespaces.rpy", line 34, in set
    setattr(self.nso, name, value)
  File "/nix/store/jrq8zj3xplxdf1fhmxhdb43pl7pq2agg-renpy-8.3.7.25031702/share/renpy/renpy/defaultstore.py", line 101, in __setattr__
    raise Exception('config.%s is not a known configuration variable.' % (name))
Exception: config.nvl_show_transition is not a known configuration variable.

Linux-6.12.34-x86_64-with-glibc2.40 x86_64
Ren'Py 8.3.7.25031702+unofficial
Honey from juniper berries 1.0
Mon Jul 14 00:57:38 2025
